assets/html/about.html

```diff
@@ -0,0 +1,176 @@
+<body>
+  <div class="container">
+    <header>
+      <a href="../../index.html">
+        <img src="../img/logonav.png" id="navlogo" alt="Code for San José logo" />
+      </a>
+      <div id="menu-btn">
+        <i onclick="menu_toggle()" id="fa-btn" class="fas fa-2x fa-bars"></i>
+      </div>
+      <div id="navlinks">
+        <div id="links">
+          <a id="homeLnk" href="../../index.html">Home</a>
+          <a id="volLnk" href="./volunteer.html">Volunteer</a>
+          <a id="abtLnk" href="./about.html">About Us</a>
+          <a id="projLnk" href="./projects.html">Projects</a>
+          <a id="evLnk" href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">Events</a>
+          <a id="donLnk"
+            href="https://secure.codeforamerica.org/page/contribute/donate-to-a-brigade-today?source_codes=Brigade-page&brigade=Code%20for%20San%20Jose"
+            target="_blank" rel="noopener noref">Donate</a>
+          <a id="faqLnk" href="./faq.html">FAQ</a>
+        </div>
+        <div id="sociallinks">
+          <a class="tooltip" href="https://codeforsanjose.slack.com/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Slack_Mark.svg" alt="Slack logo" class="socialnav" />
+            <span class="tooltiptext">Slack</span>
+          </a>
+          <a class="tooltip" href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/meetup-logo-m-swarm.png" alt="Meetup logo" class="socialnav" />
+            <span class="tooltiptext">Meetup</span>
+          </a>
+          <a class="tooltip" href="https://www.facebook.com/codeforsanjose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/f_logo_RGB-Blue_512.png" alt="Facebook logo" class="socialnav" />
+            <span class="tooltiptext">Facebook</span>
+          </a>
+          <a class="tooltip" href="https://twitter.com/codeforsanjose" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Twitter_Social_Icon_Circle_Color.svg" alt="Twitter logo" class="socialnav" />
+            <span class="tooltiptext">Twitter</span>
+          </a>
+          <a class="tooltip" href="https://github.com/codeforsanjose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/GitHub-Mark-Light-120px-plus.png" alt="Github logo" class="socialnav" />
+            <span class="tooltiptext">Github</span>
+          </a>
+          <a class="tooltip" href="https://medium.com/code-for-san-jose" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Monogram.svg" alt="Medium logo" class="socialnav" />
+            <span class="tooltiptext">Medium</span>
+          </a>
+        </div>
+      </div>
+    </header>
+
+
+    <main>
+      <div class="small-hero" id="about-title">
+        <h1>About Us</h1>
+        <div id="artist-info">
+          <p>Jonny B. Artist</p>
```

Comment:
I think this was old, maybe placeholder words? 

Here's info about the monument to include in a caption:

César E. Chávez Monument: The Arch of Dignity, Equality, and Justice
Monument by Judith F. Baca
Address: San José State University, 1 Washington Square, San José, CA 95192
This monument pays tribute to the legacy of César Chávez, Dolores Huerta, Mahatma Gandhi, and the UFW movement for dignity, equality and justice. The monument is located on El Paseo de César E. Chávez, by San José State University's Student Union.

Photo by Yan-Yin Choy

Reply:
That's a lot of information to add on an overlay for an image. Did you have a place in mind for this caption?

I did forget to replace the placeholder information, but I just pushed an update where I added the artist name, artwork title, and artist link when available.